extensions/wikia/ArticleVideo/scripts/featured-video.jwplayer.instant.js

```diff
@@ -1,96 +1,93 @@
 require([
 	'wikia.window',
+	'wikia.geo',
+	'wikia.instantGlobals',
+	'wikia.cookies',
+	'wikia.tracker',
 	'ext.wikia.adEngine.adContext',
-	'wikia.articleVideo.featuredVideo.jwplayer.instance',
 	'wikia.articleVideo.featuredVideo.data',
 	'wikia.articleVideo.featuredVideo.ads',
-	'wikia.articleVideo.featuredVideo.autoplay',
 	'wikia.articleVideo.featuredVideo.moatTracking',
-	'wikia.articleVideo.featuredVideo.tracking',
-	'wikia.articleVideo.featuredVideo.jwplayer.icons',
-	'wikia.articleVideo.featuredVideo.events',
-	'wikia.articleVideo.featuredVideo.jwplayer.logger',
-	'wikia.articleVideo.featuredVideo.jwplayer.plugin.settings',
+	'wikia.articleVideo.featuredVideo.autoplay',
 	require.optional('ext.wikia.adEngine.lookup.a9')
 ], function (
 	win,
+	geo,
+	instantGlobals,
+	cookies,
+	tracker,
 	adContext,
-	playerInstance,
 	videoDetails,
 	featuredVideoAds,
-	featuredVideoAutoplay,
 	featuredVideoMoatTracking,
-	featuredVideoTracking,
-	playerIcons,
-	featuredVideoEvents,
-	logger,
-	wikiaJWSettings,
+	featuredVideoAutoplay,
 	a9
 ) {
 	if (!videoDetails) {
 		return;
 	}
 
-	var videoId = videoDetails.mediaId,
-		inNextVideoAutoplayCountries = featuredVideoAutoplay.inNextVideoAutoplayCountries,
+	var inNextVideoAutoplayCountries = geo.isProperGeo(instantGlobals.wgArticleVideoNextVideoAutoplayCountries),
 		//Fallback to the generic playlist when no recommended videos playlist is set for the wiki
 		recommendedPlaylist = videoDetails.recommendedVideoPlaylist || 'Y2RWCKuS',
-		willAutoplay = featuredVideoAutoplay.willAutoplay,
-		pausedOnRelated = false;
+		autoplayCookieName = 'featuredVideoAutoplay',
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove the declaration of the unused 'autoplayCookieName' variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AUnusedVariable)
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this useless assignment to local variable "autoplayCookieName" [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1854)